### PR TITLE
fix: Update Controller Seeder

### PR DESCRIPTION
**Pull request by FelixMZ2018**
- Updates the Controller seeder to alwasy generate with the right display type 
- Changes the default displayTypes for slot to not include `controller parameter`
- adds a new generated connected FctGraph to the tests

Reference for the connectedPIDFctGraphData

```javascript
const { functionalitySeeders,controllerSeeder,FCTGraphUtils, FCTGraph,Functionality, FCTGraphSeeder }  = require('@ospin/fct-graph')
const{ mutators: {
  addIntervalOutFctForAllOutSlotsWhichHaveNone,
  addPushInFctForAllInSlotsWhichHaveNone,
},} = FCTGraphUtils
const temperatureSensor = functionalitySeeders.TemperatureSensorSeeder.generateCelsiusFloatProducer()
const PIDController = functionalitySeeders.PIDControllerSeeder.generateTemperatureControllerCelsius()
const heaterData = functionalitySeeders.HeaterActuatorSeeder.generateCelsiusHeater()
const fctGraph = FCTGraphSeeder.seedOne({
  functionalities: [
    temperatureSensor,
    PIDController,
    heaterData
  ]
})

const temperature = fctGraph.getFctById(temperatureSensor.id)
const pid = fctGraph.getFctById(PIDController.id)
const heater = fctGraph.getFctById(heaterData.id)


temperature.getSlotByName('Celsius Out').connectTo(pid.getSlotByName('value in'))
heater.getSlotByName('Celsius In').connectTo(pid.getSlotByName('value out'))

addIntervalOutFctForAllOutSlotsWhichHaveNone(fctGraph, { customData: { destination: { name: 'ospin-webapp' } } })
addPushInFctForAllInSlotsWhichHaveNone(fctGraph, { customData: { source: { name: 'ospin-webapp' } } })

console.log(JSON.stringify(fctGraph));
```

### CODE REVIEW

**Comment by FelixMZ2018** [comment]
> minor changes requested. Rest looks good. Not sure on a couple of things, you should wait for the Padishah Emperor @DanielSeehausen to answer

I've made all the changes requested in https://github.com/ospin-web-dev/FCTGraph/pull/99/commits/0972c1a2935e36be41bec31cb48e9942d9709d69

File: src/slots/InSlot.js [hunk 1]
```diff
@@ -9,10 +9,15 @@ class InSlot extends Slot {
...
 
+  static get CONTROLLER_PARAMETER_DISPLAY_TYPE() {
+    return 'controller parameter'
+  }
+
   static get SCHEMA() {
     return super.SCHEMA.concat(Joi.object({
       type: Joi.string().allow(InSlot.TYPE).required(),
       dataStreams: super.SCHEMA.extract('dataStreams').max(1),
+      displayType: super.SCHEMA.extract('displayType').allow(InSlot.CONTROLLER_PARAMETER_DISPLAY_TYPE),
```
[the remark this answers is not in the document]
Comment (DanielSeehausen) [reply src/slots/InSlot.js]: you flatter me with your titles, viktor
nay, i must think this package hast exceedeth me. 
the team is its steward now.

[the remark this answers is not in the document]
Comment (DanielSeehausen) [reply src/slots/InSlot.js]: (discussed in person - agreed!)

**Review by DanielSeehausen — APPROVED** [review]

**Comment by DanielSeehausen** [comment]
looks good to me! we looked in person at this guy as well

**Review by vdiezel — APPROVED** [review]

**Pull request merged**